cocos/scripting/lua-bindings/manual/cocos2d/lua_cocos2dx_manual.cpp

```diff
@@ -3812,7 +3812,7 @@ static int tolua_cocos2dx_GLProgram_setUniformLocationF32(lua_State* tolua_S)
 #endif
             
             f4 = (float)  tolua_tonumber(tolua_S,6,0);
-            if (4 == argc)
+            if (5 == argc)
```

Comment:
`if (5 == argc)` was checked before this. Is it needed here? Please refer to the logic.